tools/cli/commands/changelog.js

```diff
@@ -55,6 +55,19 @@ export function changelogDefine( yargs ) {
 					alias: 'e',
 					describe: 'Changelog entry',
 					type: 'string',
+				} )
+				.option( 'gh-action', {
+					describe: 'Output errors as github action',
+					type: 'bool',
+				} )
+				.option( 'base-dir', {
+					describe: 'Determines relative directory to show errors',
+					type: 'bool',
+				} )
+				.option( 'no-strict', {
+					alias: 'strict',
+					describe: 'Will not return failure code for only warnings',
+					type: 'bool',
```

Comment:
Adding a lot of options here since each changelog command takes differenet options. I'm not sure if there's a more concise way to do that.

Reply:
To maybe throw you into a loop... what about subcommands? Instead of having a ton of options and mush everything into a single "changelog" command, what if we break out "add", "validate", etc into commands under `changelog`?